core/src/main/groovy/noe/tomcat/configure/ConnectorAttributesTransformer.groovy

```diff
@@ -102,39 +116,69 @@ class ConnectorAttributesTransformer {
       this.connector = connector
     }
 
-    Map<String, Object> transform() {
-      Map<String, Object> res = new CommonConnectorTransformer(connector).transform()
+    Node transform() {
+      Node node = new CommonConnectorTransformer(connector).transform()
+      Map<String, Object> attributes = node.attributes()
```

Comment:
This works, but it obfuscates intention a little (that we want to change the "global" state). I would prefer to update it directly (as suggested in the previous comment) but this does not block MR for me.

Reply:
I get what you mean. You desire to act on the node and not the attributes.My change is just a stage before because I haven't it figured out

Reply:
Yes, correct - you named it as I thought.